WalletWasabi/Wabisabi/IssuanceValidationData.cs

```diff
@@ -0,0 +1,22 @@
+﻿using NBitcoin;
+using NBitcoin.Secp256k1;
+using WalletWasabi.Crypto.Groups;
+
+namespace WalletWasabi.Wabisabi
+{
+	public class IssuanceValidationData
```

Comment:
this class is somewhat redundant with `Attribute`, the only real difference being that this has an `Amount` property, so I think this class could be eliminated

Reply:
It is not redundant anymore.